test/acceptance/renumber.js

```diff
@@ -0,0 +1,12 @@
+describe('todo renumber', function() {
```

Comment:
`Todo`, `Todos`, `Storage`. Fancy way of saying the things your app is made up of.

So ya, it would be all the operations on the  [`Storage`](https://github.com/vesln/todo/blob/master/lib/storage.js) object or the [`Todos`](https://github.com/vesln/todo/blob/master/lib/todos.js) object.